app/ts/background/settings.ts

```diff
@@ -131,7 +131,8 @@ export async function getSettings() : Promise<Settings> {
 	}
 }
 
-export const setPage = async (page: Page) => await browserStorageLocalSet({page})
+export const setPage = async (openedPage: Page) => await browserStorageLocalSet({ openedPage })
+export const getPage = async() => (await browserStorageLocalGet('openedPage'))?.['openedPage'] ?? { page: 'Home' }
```

Comment:
doesn't the first condition already coalesce as null?

```suggestion
export const getPage = async() => (await browserStorageLocalGet('openedPage')) ?? { page: 'Home' }
```

Reply:
yeah, but then it returns `{ openedPage: { page: 'home' } }` on success, instead of the wanted `{ page: 'home' }` field